dev-db/firebird/firebird-2.5.3.26780.0-r3.ebuild

```diff
@@ -20,7 +20,7 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
 IUSE="debug doc client examples superserver xinetd"
-REQUIRED_USE="^^ ( client superserver xinetd )"
+REQUIRED_USE="?? ( client superserver xinetd )"
```

Comment:
The change that introduced the ^^ ( ) logic replaced this, it did effectively the same:
```
REQUIRED_USE="
	client? ( !superserver )
	client? ( !xinetd )
	superserver? ( !xinetd )
"
```